docs/sources/reference/commandline/cli.md

```diff
@@ -1608,8 +1628,8 @@ deleted.
 
 #### Removing tagged images
 
-Images can be removed either by their short or long IDs, or their image
-names. If an image has more than one name, each of them needs to be
+An image can be removed by its short or long ID, its tag, or its digest. If an
```

Comment:
@moxiegirl when you remove the last reference (tag or digest) to an image, the image data is removed. There isn't an additional `rmi` operation that you must perform after you've removed all of the references.